.github/release-drafter.yml

```diff
@@ -0,0 +1,28 @@
+name-template: 'v$NEXT_PATCH_VERSION'
```

Comment:
@adithyaakrishna it's not obvious, but we now have a convention of adding the project name like so:

`name-template: 'Meshery Adapter for App Mesh v$NEXT_PATCH_VERSION'`

Reply:
@leecalcote My bad, I had missed that. I've registered the convention in my brain now, will keep this in mind while working on future issues 💯